### PR TITLE
zic, tzselect: add page 

### LINKED ISSUE #14018: Let's document: tzdata
### Command description

Handle timezone data.

### Documentation

https://www.iana.org/time-zones

### Platform

Linux

### VCS repository link (e.g. GitHub, GitLab)

_No response_

### Additional information

_No response_

### Commands

- [x] zdump https://github.com/tldr-pages/tldr/pull/14017
- [ ] zic https://github.com/tldr-pages/tldr/pull/14312
- [ ] tzselect https://github.com/tldr-pages/tldr/pull/14312

### PR DESCRIPTION
**Pull request by angadsgrover**
Resolves #14018 

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known): 2024b**

### CODE REVIEW

**Review by gutjuri — APPROVED** [review]
Thanks!

File: pages/linux/tzselect.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# tzselect
+
+> Interactively select timezone.
+> More information: <https://manned.org/tzselect>.
+
+- Open the interactive menu for timezone selection:
+
+`tzselect`
```
Comment (acuteenvy) [inline pages/linux/tzselect.md:8]: This program doesn't actually set the timezone, and the current description kind of implies that it does.

```suggestion
> Interactively select a timezone.
> Note: this program doesn't actually set the timezone.
> More information: <https://manned.org/tzselect>.

- Open the interactive menu for timezone selection and print the selected timezone to `stdout`:

`tzselect`
```

File: pages/linux/zic.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# zic
+
+> Compile timezones into binary files.
+> More information: <https://manned.org/zic>.
+
+- Compile a timezone file from directory:
+
+`zic -d {{path/to/file}}.infile`
```
Comment (acuteenvy) [inline pages/linux/zic.md:8]:
```suggestion
- Compile a timezone file from a directory:

`zic -d {{path/to/directory}}`
```

File: pages/linux/zic.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+- Report warnings during compilation of a specific file:
+
+`zic -v {{path/to/file}}.infile`
```
Comment (acuteenvy) [inline pages/linux/zic.md:12]:
```suggestion
- Report warnings during compilation of a specific file:

`zic -v {{path/to/file}}`
```

**Review by acuteenvy — APPROVED** [review]

**Review by spageektti — APPROVED** [review]

File: pages/linux/zic.md (hunk 2 above)
Comment (angadsgrover) [reply pages/linux/zic.md:8]: We have to select a file via the path for the command as per the description, so should it be changed to path/to/directory?

**Pull request merged**